editor/icons/icon_cylinder_shape.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="16" height="16" version="1.1" viewBox="0 0 14.999999 14.999999" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Why "14.999999" instead of "15"?

Reply:
Indeed as mentioned I should fix the icon, I hastily traced it in inkscape from either icon_c_s_g_cylinder.svg or icon_cylinder_mesh.svg then cleaned it manually, but those where paths! should this three shapes also be converted to paths?
Can i just keep the separated shapes and round the values?

I was thinking of fixing that if the rest of the PR was ok and acceptable